### PR TITLE
homework-week3

### PR DESCRIPTION
**Pull request by viedamayanti**

### CODE REVIEW

File: week3/homework-week3/Program.cs
Comment (calindurnea) [inline week3/homework-week3/Program.cs:1]: You can use the formatting functionality from Visual studio code to make the code easier to read (`Alt+Shift+F` on windows)

File: week3/homework-week3/Program.cs [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        return result;
+    }
+}
+
+
+//Bank account
+public class Account
+{
+   public decimal Amount{ get; set; }
+    public decimal Balance { get; set;}
```
Comment (LaurentiuCalin) [inline week3/homework-week3/Program.cs:98]: this should be a `get` only property 😊

File: week3/homework-week3/Program.cs [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+    public Account(decimal balance)
+    {
+        Balance = balance ;
+
+    }
+         public decimal Deposit(decimal Amount)
+    {
+        {
+            if (Balance > 0)
```
Comment (calindurnea) [inline week3/homework-week3/Program.cs]: There is no need to check here if the Balance is positive as this is checked on withdrawal

File: week3/homework-week3/Program.cs [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+            {
+                Balance += Amount;
+            }
+            return Balance;
+        }
+    }
+
+     public decimal Withdraw(decimal Amount)
+     {
+     {
```
Comment (calindurnea) [inline week3/homework-week3/Program.cs]: it is not necessary to have the curly brackets two times here and in the `Deposit` method

File: week3/homework-week3/Program.cs [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+        if (Balance <= Amount)
```
Comment (calindurnea) [inline week3/homework-week3/Program.cs]: I think that if the Balance and the Amount are of equal value you should still be allowed to withdraw as you will be on 0 and not negative balance

File: week3/homework-week3/Program.cs [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+
+    }
+         public decimal Deposit(decimal Amount)
+    {
+        {
+            if (Balance > 0)
+            {
+                Balance += Amount;
+            }
+            return Balance;
```
Comment (calindurnea) [inline week3/homework-week3/Program.cs]: You can as well remove the return from the method as the value of `Balance` is updated above and you don't use it in the code above. (Same with the Withdraw method)

**Review by calindurnea — APPROVED** [review]
Great job🥇 ! I have left some comments, please reach out if you have any questions 👍

**Comment by viedamayanti** [comment]
Thank you so much for feedback 🙏

**Pull request merged**